base-template/about-base-template.md

```diff
@@ -1,45 +1,63 @@
+The base template guides you through the steps to create a functional template that can be reused, adhered to, and followed consistently throughout.
 
-## Content of your template
+However, note that the base template cannot be used to arrive at the document type you need, and is not a substitute for any other template in the collection at the main repository. 
 
-### About the "Prerequisites" section
+## Prerequisites
 
-Professionally communicate high standards in total linkage via clicks-and-mortar relationships. Interactively predominate enabled metrics before distinctive best practices. Distinctively visualize parallel networks before cutting-edge partnerships. Uniquely synthesize multidisciplinary quality vectors rather than multidisciplinary scenarios. Conveniently drive team driven products through empowered alignments.
+To make the best use of this base template, you need a working knowledge of:
 
-### About the "Overview" section
+- GitHub to download, edit, and use the base template.
+- Markdown to create and edit the base template to suit your requirements.
+- Different documentation types to identify if you need to create a new template or edit an existing one.
+- Technical writing to produce a well-written document that includes the relevant reference aspects.
 
-Professionally redefine interactive deliverables whereas client-based e-commerce. Uniquely impact competitive quality vectors after resource-leveling content. Competently repurpose economically sound value before strategic processes. Competently benchmark global action items rather than user friendly internal or "organic" sources. Continually plagiarize flexible e-business after strategic core competencies.
+## Who will need the base template
 
-### About the content
+The base template is for:
 
-Energistically embrace worldwide applications without customized growth strategies. Energistically embrace orthogonal convergence vis-a-vis global internal or "organic" sources. Dramatically transform leading-edge networks rather than client-based convergence. Credibly revolutionize web-enabled interfaces before strategic solutions. Authoritatively incubate economically sound e-business with equity invested outsourcing.
+- Internal users - those who are already working in the Good Docs Project (GDP)
+- External users - the community outside of the GDP
 
-### About the "Reference" section
+Who want to:
 
-Professionally cultivate emerging alignments through multimedia based initiatives. Dramatically administrate cross-media users rather than cooperative paradigms. Synergistically redefine long-term high-impact niches via cross functional relationships. Authoritatively whiteboard interactive "outside the box" thinking with dynamic supply chains. Assertively pontificate standardized applications before goal-oriented sources.
+- Create new templates that do not yet exist within GDP
```

Comment:
I personally try to avoid using acronyms such as "GDP", especially for new users, as it slows their comprehension.
If you do use GDP, it should be "the GDP".

Reply:
Noted. Will correct accordingly.